source/agora/common/Serializer.d

```diff
@@ -67,18 +68,57 @@ private enum hasSerializeMethod (T) = is(T == struct)
 
 public ubyte[] serializeFull (T) (scope const auto ref T record)
     @safe
-    if (is(T == struct))
```

Comment:
Will this end up enabling serialization for classes too?

Reply:
Hum yes although that wasn't the point.

Reply:
[Aaaaaaaaaacccckkkkktually...](https://github.com/bpfkorea/agora/blob/c0b4e7ba5e72dafab168238439e09df3b5b3e37d/source/agora/common/BanManager.d#L266-L311)